filecoin-proofs/src/api/update.rs

```diff
@@ -60,6 +60,12 @@ impl SectorUpdateProofInputs {
         dest[33..64].copy_from_slice(&self.comm_r_new[..]);
         dest[65..96].copy_from_slice(&self.comm_d_new[..]);
     }
+
+    pub fn update_commitment(&self, hasher: &mut Sha256) {
```

Comment:
Doesn't seem that it needs to be `pub`.

Also I would directly put it into `get_hashed_commitments`, but that's really a matter of taste, so it's totally fine to have it in a separate function.